conf/script/build-ha-csm

```diff
@@ -159,7 +163,7 @@ systemd_disable() {
 kibana_rsc_add() {
     echo 'Adding kibana resources...'
     sudo pcs -f $cib_file resource create kibana-vip ocf:heartbeat:IPaddr2 \
-        ip=$vip cidr_netmask=24 nic=$iface iflabel=v1 \
+        ip=$vip cidr_netmask=$cidr nic=$iface iflabel=v1 \
```

Comment:
Don't we need this for data ips too?

Reply:
@mssawant 
Data IPs are handled by Provisioner.
On another note, do you expect Provisioner to provide/pass cidr value?

In Provisioner code, it's queried from existing network config:
https://github.com/Seagate/cortx-prvsnr/pull/741/files#diff-abd1c58819b8dae179a4aab54e259bdf59c1e413c8d29f8cd4e666a5afd1ba81R35

Reply:
Hi @mssawant. Talking about ip-c1/2 resources - the netmask is determined automatically by build-ha-io script depending on existing IP address on the interfaces. Private data interface has static IP (probably configured by provisioner). So i think we are good here so far.
I think the issue is about ClusterIP which indeed handled by Provisioner.
@ypise yes, we are expecting Provisioner to pass netmask value for kibana-vip resource, otherwise default /24 netmask will be set.